custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -37,22 +38,21 @@ def __init__(self):
         self._mlops_spool_dir = None
 
     def mlpiper_configure(self, params):
+        target_type = TargetType(params.get("target_type"))
+        code_dir = params["__custom_model_path__"]
+
+        self._model_adapter = PythonModelAdapter(model_dir=code_dir, target_type=target_type)
+
+        sys.path.append(code_dir)
+        self._model_adapter.load_custom_hooks()
+
```

Comment:
Intialize model adapter before calling `super()` so that we can check if model adapter has chat hook when `_supports_chat` is called from super.